spec/requests/users_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+  describe "GET /index" do
+    it "returns http success" do
+      get "/users"
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET /show" do
+    it "returns http success" do
+      user = User.create!(first_name: "Rodrigo", email: "rcarmona@example.com", password: "password")
+      get "/users/#{user.id}"
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET /create" do
+    it "returns http success" do
+      post "/users", params: { user: { first_name: "Gonzalo", last_name: "Carmona", email: "gcarmona@example.com", password: "password" } }
+      expect(response).to have_http_status(:redirect)
+    end
+  end
+
+  describe "GET /update" do
```

Comment:
This one is not correct.
Either create a describe for **`"PUT /update`** or create a describe for **`"GET /edit"`** (or both)